<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget { background: rgba(32, 80, 96, 100); }
QLabel 
{ 
color: white; 
background: rgba(32, 80, 96, 100);
}
QLineEdit 
{ 
border-radius: 3px;
background-color: rgb(194, 196, 201);
}
QPlainTextEdit
{
border-radius: 3px;
background-color: rgb(194, 196, 201);
}
QPushButton
{
color: white;
background-color: #27a9e3;
border-width: 0px;
border-radius: 3px;
}
QPushButton:hover {background-color: rgb(154, 153, 221);}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLineEdit" name="IpEdit">
      <property name="text">
       <string>127.0.0.1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QPushButton" name="StartHandshakeBtn">
      <property name="text">
       <string>Start Handshake</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QPushButton" name="SendDataBtn">
      <property name="text">
       <string>Send Data</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="PortEdit">
      <property name="text">
       <string>4444</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="ConnectionState">
      <property name="text">
       <string>Connection State:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QPlainTextEdit" name="Log">
      <property name="font">
       <font>
        <family>Courier 10 Pitch</family>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="ConnecntionStateLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLineEdit" name="SendDataEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
